CHANGELOG.md

```diff
@@ -18,6 +18,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   left after children were wrapped. [#930]
 - FSDP: Add support for saving optimizer state when using expert replicas with FSDP.
 
+### Fixed
+- FSDP: fixing handling of internal states with state_dict and load_state_dict.
```

Comment:
```suggestion
- FSDP: Fixed handling of internal optimizer state using state_dict and load_state_dict to support the unused parameters use case.
```

Reply:
Actually, this part is not about optimizer state, this is about state_dict function. I will make it more clear.